botorch_community/posteriors/bll_posterior.py

```diff
@@ -9,31 +9,31 @@
 import math
 
 import torch
-from botorch.posteriors import GPyTorchPosterior, Posterior
-
+from botorch.posteriors import GPyTorchPosterior
 from botorch_community.models.blls import AbstractBLLModel
+from gpytorch.distributions import MultivariateNormal
 
 from torch import Tensor
 
 
-class BLLPosterior(Posterior):
+class BLLPosterior(GPyTorchPosterior):
     def __init__(
         self,
-        posterior: GPyTorchPosterior,
         model: AbstractBLLModel,
+        mvn_dist: MultivariateNormal,
         X: Tensor,
         output_dim: int,
     ):
         """A posterior for Bayesian last layer models.
 
         Args:
-            posterior: A posterior object.
             model: A BLL model
+            mvn_dist: MultivariantNormal distribution for the posterior.
             X: Input data on which the posterior was computed.
             output_dim: Output dimension of the model.
         """
-        super().__init__()
-        self.posterior = posterior
+        super().__init__(mvn_dist)
+        self.mvn_dist = mvn_dist
```

Comment:
this seems unnecessary since now with subclassing from `GPyTorchPosterior` `self.distribution` already contains this?